Route rate limits (sec #18) (#2530)
flowforge/flowforge: docs/install/configuration.md

`telemetry.frontend.posthog.capture_pageview` | FlowForge is designed as to provide custom posthog `$pageview` events that provide more detail on navigation than the default, and suit a single page application better. As such, we recommend setting this to false in order to prevent duplicate `pageleave`/`pageview` events firing. Default: `true`

## Rate Limiting configuration

By default, rate limiting is disabled and the platform will not rate limit any requests.

To enable rate limiting, you can set the `rate_limits.enabled` option to `true`. This will enable rate limiting for all routes unless explicitly disabled (see note below).

> By default, all routes will be limited requests to 1000 per 1 minute window for logged in users, and 60 per 1 minute window for anonymous users.

Author: ```suggestion
By default, all routes will be limited to 1000 per 1 minute window.
```

> By default, rate limiting is disabled and the platform will not rate limit any requests.
> 
> To enable rate limiting, you can set the `rate_limits.enabled` option to `true`. This will enable rate limiting for all routes unless explicitly disabled (see note below).
> By default, all routes will be limited requests to 1000 per 1 minute window for logged in users, and 60 per 1 minute window for anonymous users.

Author: ```suggestion
By default, rate limiting is disabled and the platform will not rate limit any requests.

To enable rate limiting, you can set the `rate_limits.enabled` option to `true`. 

When enabled, all routes will be limited to 1000 requests per 1 minute window. These defaults can be adjusted by setting values in the configuration options listed below.
```

`telemetry.frontend.posthog.capture_pageview` | FlowForge is designed as to provide custom posthog `$pageview` events that provide more detail on navigation than the default, and suit a single page application better. As such, we recommend setting this to false in order to prevent duplicate `pageleave`/`pageview` events firing. Default: `true`

## Rate Limiting configuration

By default, rate limiting is disabled and the platform will not rate limit any requests.

To enable rate limiting, you can set the `rate_limits.enabled` option to `true`. 

When enabled, all routes will be limited to 1000 requests per 1 minute window. These defaults can be adjusted by setting values in the configuration options listed below.

> NOTE: Some routes are prohibited from being rate limited, such as the logout and acl routes.

Reviewer: While I understand why this is documented I wonder if it just makes it a target for attack

Author: Good question Ben.  Open to suggestions.

> When enabled, all routes will be limited to 1000 requests per 1 minute window. These defaults can be adjusted by setting values in the configuration options listed below.
> 
> NOTE: Some routes are prohibited from being rate limited, such as the logout and acl routes.

Author: ```suggestion
```

@hardillb - remove this section?

Something else?

Author: will re-do.

> NOTE: Some routes are prohibited from being rate limited, such as the logout and acl routes.

Author: @hardillb 🤞
